src/main/java/teammates/ui/controller/AjaxResult.java

```diff
@@ -36,7 +36,8 @@ public AjaxResult(AccountAttributes account,
     @Override
     public void send(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
         
-        req.setAttribute(Const.ParamsNames.ERROR, "" + isError);        
+        req.setAttribute(Const.ParamsNames.ERROR, "" + isError);
```

Comment:
Const.ParamsNames.ERROR was previously used to colour the status messages red, is it still used?